### ISSUE TITLE
Can’t attach files to cards anymore

### ISSUE THREAD

**Issue by octavsly** [nextcloud/deck#6009]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Describe the bug**
https://help.nextcloud.com/t/cant-attach-files-to-cards-anymore/193993
NC 29.0.0, Deck 1.13.0

Try to add an attachment by sharing from files, the file picker dialog is never populated, so I can’t pick anything. Seems to be a Deck issue, as the dialog works in other apps.

What I noticed is that I get a “TypeError: t is undefined” in the browser console, at main.js:56:9. This doesn’t seem really helpful, as it is just an error handler, and t is supposed to be the translate method.

**To Reproduce**
1. Go to 'a deck card'
2. Click on 'Attachements'
3. Click on 'Share from Files'
4. 'file picker dialog is never populated'

**Expected behavior**
 'file picker dialog should show all the files/directories'

**Screenshots**
![image](https://github.com/nextcloud/deck/assets/818628/c8e03bde-ad39-4a61-aa16-90fb0609938b)

**Client details:**
 - OS: [Linux Gentoo]
 - Browser [firefox]
 - Version [126.0.1]
 - Device: [desktop]

<details>
<summary>Server details</summary>

**Operating system**:
Linux 

**Web server:**
apache 2.4.59

**Database:**
mysql 

**PHP version:**
8.2

**Nextcloud version:** (see Nextcloud admin page)
29.0.2

**Where did you install Nextcloud from:**
gentoo distribution 

**Signing status:**

```
Login as admin user into your Nextcloud and access
http://example.com/index.php/settings/integrity/failed
paste the results here.

No errors have been found.
```

**List of activated apps:**

```
Enabled:
  - activity: 2.21.1
  - admin_audit: 1.19.0
  - bruteforcesettings: 2.9.0
  - calendar: 4.7.6
  - circles: 29.0.0-dev
  - cloud_federation_api: 1.12.0
  - comments: 1.19.0
  - contactsinteraction: 1.10.0
  - dashboard: 7.9.0
  - dav: 1.30.1
  - deck: 1.13.0
  - documentserver_community: 0.1.17
  - emlviewer: 1.0.9
  - federatedfilesharing: 1.19.0
  - federation: 1.19.0
  - files: 2.1.0
  - files_downloadlimit: 2.0.0
  - files_external: 1.21.0
  - files_linkeditor: 1.1.20
  - files_pdfviewer: 2.10.0
  - files_reminders: 1.2.0
  - files_sharing: 1.21.0
  - files_trashbin: 1.19.0
  - files_versions: 1.22.0
  - firstrunwizard: 2.18.0
  - logreader: 2.14.0
  - lookup_server_connector: 1.17.0
  - mail: 3.7.1
  - memories: 7.3.1
  - nextcloud_announcements: 1.18.0
  - notifications: 2.17.0
  - oauth2: 1.17.0
  - onlyoffice: 9.2.2
  - password_policy: 1.19.0
  - polls: 7.1.1
  - previewgenerator: 5.5.0
  - privacy: 1.13.0
  - provisioning_api: 1.19.0
  - recommendations: 2.1.0
  - related_resources: 1.4.0
  - serverinfo: 1.19.0
  - settings: 1.12.0
  - sharebymail: 1.19.0
  - spreed: 19.0.2
  - support: 1.12.0
  - survey_client: 1.17.0
  - suspicious_login: 7.0.0
  - systemtags: 1.19.0
  - text: 3.10.0
  - theming: 2.4.0
  - twofactor_backupcodes: 1.18.0
  - updatenotification: 1.19.1
  - user_status: 1.9.0
  - viewer: 2.3.0
  - weather_status: 1.9.0
  - workflowengine: 2.11.0
Disabled:
  - contacts: 6.0.0 (installed 6.0.0)
  - encryption: 2.17.0
  - groupfolders: 17.0.1 (installed 17.0.1)
  - integration_whiteboard: 0.0.14 (installed 0.0.14)
  - photos: 2.5.0 (installed 2.4.0)
  - twofactor_totp: 11.0.0-dev
  - user_ldap: 1.20.0

If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

**Nextcloud configuration:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "nextcloud.indenaamvanChristus.eu"
        ],
        "default_phone_region": "NL",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "29.0.2.2",
        "overwrite.cli.url": "https:\/\/nextcloud.indenaamvanChristus.eu",
        "overwriteprotocol": "https",
        "htaccess.RewriteBase": "\/",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "sendmail",
        "mail_sendmailmode": "pipe",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "25",
        "maintenance": false,
        "maintenance_window_start": 1,
        "filelocking.enabled": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "dbindex": 0,
            "timeout": 1.5,
            "password": "***REMOVED SENSITIVE VALUE***"
        },
        "theme": "",
        "loglevel": 0,
        "simpleSignUpLink.shown": false,
        "enable_previews": true,
        "enabledPreviewProviders": {
            "90": "OC\\Preview\\Postscript",
            "0": "OC\\Preview\\TXT",
            "1": "OC\\Preview\\MarkDown",
            "3": "OC\\Preview\\PDF",
            "6": "OC\\Preview\\HEIC",
            "7": "OC\\Preview\\Image",
            "8": "OC\\Preview\\Photoshop",
            "9": "OC\\Preview\\TIFF",
            "10": "OC\\Preview\\SVG",
            "11": "OC\\Preview\\Font",
            "12": "OC\\Preview\\MP3",
            "13": "OC\\Preview\\Movie",
            "14": "OC\\Preview\\MKV",
            "15": "OC\\Preview\\MP4",
            "16": "OC\\Preview\\AVI",
            "17": "OC\\Preview\\AAC"
        },
        "has_rebuilt_cache": true,
        "app_install_overwrite": [
            "emlviewer",
            "whiteboard",
            "integration_whiteboard",
            "spreed",
            "documentserver_community",
            "onlyoffice"
        ],
        "memories.exiftool_no_local": true,
        "memories.vod.path": "\/var\/www\/indenaamvanChristus.ga\/htdocs\/nextcloud\/apps\/memories\/bin-ext\/go-vod-amd64",
        "memories.vod.ffmpeg": "\/usr\/bin\/ffmpeg",
        "memories.vod.ffprobe": "\/usr\/bin\/ffprobe",
        "memories.db.triggers.fcu": true
    }
}

```

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

</details>

**Comment by StefInP** [comment]
It seems to be a duplicate with #5822

**Comment by niclasheinz** [comment]
With the update of Nextcloud Deck to 1.13.1 this error has disappeared.

**Comment by DecaTec** [comment]
> With the update of Nextcloud Deck to 1.13.1 this error has disappeared.

afaik 1.13.1 was released for NC 29. \
For NC 28, version 1.12.3 seems to be the current one and here the fix from 1.13.1 is still mising. 

Any chance that this is also fixed for NC 28?

**Comment by niclasheinz** [comment]
I'll check that and Inform you.

**Comment by niclasheinz** [comment]
Unfortunately there will be no fix for 28 and 27 (see https://github.com/nextcloud/deck/pull/6038#issuecomment-2206006635)

**Comment by elzody** [comment]
I have a fix awaiting approval for NC 28 here: #6092

@niclasheinz @DecaTec

**Comment by juliushaertl** [comment]
Just published a new release with the fix included https://github.com/nextcloud-releases/deck/releases/tag/v1.12.4
